test_scripts/Defects/4_5/1881_OnDriverDistraction_After_changing_HMIlevel_from_NONE.lua

```diff
@@ -0,0 +1,114 @@
+
+local function ActivationAppWithOnDD( params, self)
+  local hmiAppId = commonDefects.getHMIAppId(1)
+  local requestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = hmiAppId })
+  EXPECT_HMIRESPONSE(requestId)
+  self.mobileSession1:ExpectNotification("OnHMIStatus",
+    { hmiLevel = "FULL", audioStreamingState = "AUDIBLE", systemContext = "MAIN" })
+  self.mobileSession1:ExpectNotification("OnDriverDistraction", params)
+end
+
+local function RegisterAppWithOnDDResumption( params, self)
+  self.mobileSession1 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession1:StartService(7)
+  :Do(function()
+      local corId = self.mobileSession1:SendRPC("RegisterAppInterface", config.application1.registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered",
+        { application = { appName = config.application1.registerAppInterfaceParams.appName } })
+      self.mobileSession1:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+      :Do(function()
+          self.mobileSession1:ExpectNotification("OnHMIStatus",
+            { hmiLevel = "NONE" },
+            { hmiLevel = "LIMITED" })
+          :Times(2)
+          self.mobileSession1:ExpectNotification("OnPermissionsChange")
+          self.mobileSession2:ExpectNotification("OnDriverDistraction")
+          :Times(0)
+          self.mobileSession1:ExpectNotification("OnDriverDistraction", params)
+          commonDefects.delayedExp(1000)
+        end)
+    end)
+end
+
+local function ActivationAppWithOnDD2Apps( params, self)
+  local hmiAppId = commonDefects.getHMIAppId(2)
+  local requestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = hmiAppId })
+  EXPECT_HMIRESPONSE(requestId)
+  self.mobileSession2:ExpectNotification("OnHMIStatus",
+    { hmiLevel = "FULL"})
+  self.mobileSession2:ExpectNotification("OnDriverDistraction", params)
+  self.mobileSession1:ExpectNotification("OnDriverDistraction")
+  :Times(0)
+  self.mobileSession1:ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED"})
+  commonDefects.delayedExp(1000)
+end
+
+local function CloseSession(self)
+  self.mobileSession1:Stop()
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonDefects.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonDefects.start)
+runner.Step("RAI, PTU", commonDefects.rai_ptu)
+
+runner.Title("Test")
+runner.Step("Absence_OnDriverDistraction_in_NONE", OnDDinNONE, {1, {state = "DD_ON"}})
+runner.Step("OnDriverDistraction_in_FULL", ActivationAppWithOnDD, {{state = "DD_ON"}})
+runner.Step("UnregisterRegisterApp", commonDefects.unregisterApp, {1})
+
+runner.Step("RAI_first_app", commonDefects.rai_n, {1})
+runner.Step("RAI_second_app", commonDefects.rai_ptu_n, {2})
+runner.Step("Activate_first_app", commonDefects.activate_app)
+runner.Step("OnDriverDistraction_in_FULL_ansence_in_NONE", OnDD2Apps, {1, 2, {state = "DD_ON"}})
```

Comment:
ansence -> absence

Reply:
@LitvinenkoIra, updated, please find update in 91c6f035bcaa364dc87d2cd812f7c4fd2841de15.